astacus/common/cassandra/config.py

```diff
@@ -29,19 +29,19 @@
 
 
 class CassandraClientConfiguration(AstacusModel):
-    config_path: Optional[Path]
+    config_path: Path | None = None
 
     # WhiteListRoundRobinPolicy contact points
-    hostnames: Optional[List[str]]
```

Comment:
this requires python larger then 3.10. Probably it is not stated in required packages

Reply:
It's already specified 3.11 in all places: https://github.com/Aiven-Open/astacus/commit/55c97cb0ba41159c004327974110943187ef5ec4